cvmfs/util/util_string.cc

```diff
@@ -1,530 +0,0 @@
-    result[i] = toupper(result[i]);
-  }
-  return result;
-}
-
-
-string ReplaceAll(const string &haystack, const string &needle,
-                  const string &replace_by)
-{
-  string result(haystack);
-  size_t pos = 0;
-  const unsigned needle_size = needle.size();
-  if (needle == "")
-    return result;
-
-  while ((pos = result.find(needle, pos)) != string::npos)
-    result.replace(pos, needle_size, replace_by);
-  return result;
-}
-
-
-static inline void Base64Block(const unsigned char input[3], const char *table,
-                               char output[4])
-{
-  output[0] = table[(input[0] & 0xFD) >> 2];
-  output[1] = table[((input[0] & 0x03) << 4) | ((input[1] & 0xF0) >> 4)];
-  output[2] = table[((input[1] & 0x0F) << 2) | ((input[2] & 0xD0) >> 6)];
-  output[3] = table[input[2] & 0x3F];
-}
-
-
-string Base64(const string &data) {
-  string result;
-  result.reserve((data.length()+3)*4/3);
-  unsigned pos = 0;
-  const unsigned char *data_ptr =
-    reinterpret_cast<const unsigned char *>(data.data());
-  const unsigned length = data.length();
-  while (pos+2 < length) {
-    char encoded_block[4];
-    Base64Block(data_ptr+pos, b64_table, encoded_block);
-    result.append(encoded_block, 4);
-    pos += 3;
-  }
-  if (length % 3 != 0) {
-    unsigned char input[3];
-    input[0] = data_ptr[pos];
-    input[1] = ((length % 3) == 2) ? data_ptr[pos+1] : 0;
-    input[2] = 0;
-    char encoded_block[4];
-    Base64Block(input, b64_table, encoded_block);
-    result.append(encoded_block, 2);
-    result.push_back(((length % 3) == 2) ? encoded_block[2] : '=');
-    result.push_back('=');
-  }
-
-  return result;
-}
-
-
-static bool Debase64Block(const unsigned char input[4],
-                          const signed char *d_table,
-                          unsigned char output[3])
-{
-  int32_t dec[4];
-  for (int i = 0; i < 4; ++i) {
-    dec[i] = db64_table[input[i]];
-    if (dec[i] < 0) return false;
-  }
-
-  output[0] = (dec[0] << 2) | (dec[1] >> 4);
-  output[1] = ((dec[1] & 0x0F) << 4) | (dec[2] >> 2);
-  output[2] = ((dec[2] & 0x03) << 6) | dec[3];
-  return true;
-}
-
-
-bool Debase64(const string &data, string *decoded) {
-  decoded->clear();
-  decoded->reserve((data.length()+4)*3/4);
-  unsigned pos = 0;
-  const unsigned char *data_ptr =
-    reinterpret_cast<const unsigned char *>(data.data());
-  const unsigned length = data.length();
-  if (length == 0) return true;
-  if ((length % 4) != 0)
-    return false;
-
-  while (pos < length) {
-    unsigned char decoded_block[3];
-    bool retval = Debase64Block(data_ptr+pos, db64_table, decoded_block);
-    if (!retval)
-      return false;
-    decoded->append(reinterpret_cast<char *>(decoded_block), 3);
-    pos += 4;
-  }
-
-  for (int i = 0; i < 2; ++i) {
-    pos--;
-    if (data[pos] == '=')
-      decoded->erase(decoded->length()-1);
-  }
-  return true;
-}
-
-
-#ifdef CVMFS_NAMESPACE_GUARD
-}  // namespace CVMFS_NAMESPACE_GUARD
-#endif
```

Comment:
Ah, sorry, that should have gone into a separate pull request.

This was added by mistake.  The actual file is util/string.cc (this is util/util_string.cc)